packages/next-codemod/transforms/name-default-component.ts

```diff
@@ -29,13 +29,30 @@ export default function transformer(
 
   let hasModifications: boolean
 
-  const returnsJSX = (node): boolean =>
-    node.type === 'JSXElement' ||
-    (node.type === 'BlockStatement' &&
-      j(node)
-        .find(j.ReturnStatement)
-        .some((path) => path.value.argument?.type === 'JSXElement'))
-
+  const returnsJSX = (node): boolean => {
+    try {
+      if (!node) {
+        return false
+      } else {
+        return (
+          node?.type === 'JSXElement' ||
+          (node?.type === 'BlockStatement' &&
+            j(node)
+              .find(j.ReturnStatement)
+              .some((path) => {
+                return (
+                  path?.value?.argument?.type === 'JSXElement' ||
+                  // @ts-ignore
+                  path?.argument?.type === 'JSXElement'
```

Comment:
this I might wanna cleanup, but I was seeing the possibility of just argument.type here when looking on AST Exploer